civihr_default_theme/scss/base/_mixins.scss

```diff
@@ -181,3 +181,9 @@
   background-repeat: no-repeat;
   background-position: right center;
 }
+
+@mixin views-label {
+  color: $gray-darker;
+  font-size: 1.3rem;
```

Comment:
not 100% sure, but i don't think we are using `rem` in this theme

Reply:
Ok, changed to a variable which contains font-size in pixels.